justfile

```diff
@@ -127,15 +121,20 @@ generate cli=default_cli platform=default_platform:
         ;;
     esac
 
+# Apply the runtime.
+apply-runtime target=default_deploy_target platform=default_platform:
+    #!/usr/bin/env bash
+    set -euo pipefail
+    kubectl apply -f ./{{ workspace_dir }}/runtime
+    if [[ {{ platform }} == "AKS-PEER-SNP" ]]; then
+        kubectl apply -f ./infra/azure-peerpods/peer-pods-config.yaml --namespace {{ target }}${namespace_suffix-}
```

Comment:
We are still unable to deploy the runtime without knowing the target namespace, right? Please create a ticket with a detailed issue description.

Reply:
No, this has been fixed. Instead of using federated credentials, we know use a application password credential and directly pass a client id and secret to the cloud-api-adaptor. Conventionally, we already had a application password credential set up:
https://github.com/edgelesssys/contrast/blob/0362d8ccc7892311a799dfca717ac84cc595cea9/infra/azure-peerpods/main.tf#L80C11-L82